services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeYhteystietoEntity.kt

```diff
@@ -136,6 +141,7 @@ class HankeYhteystietoEntity(
 
 @Schema(description = "Contact person")
 data class Yhteyshenkilo(
```

Comment:
Should this class be in separate file or in `HankeYhteyshenkiloEntity.kt`? Also, method `toHankekayttajaInput()` seems to be unnecessary.

Reply:
I moved this to a separate class under domain-package. I've been thinking about extracting the yhteystieto stuff from HankeService to a new service and creating a new package around that with the entities and domain classes, but I haven't found the right time to do that.